packages/slonik/src/sqlFactory.ts

```diff
@@ -78,8 +79,11 @@ const SqlFactory = <T extends QueryResultRow, I extends QueryResultRow>(
       return sql<T>`
         SELECT *
         FROM ${createTableFragment(tableName, schema)}
-        ${createFilterFragment(filters, tableName)}
-        ${createSortFragment(tableName, sort)}
+        ${createFilterFragment(
+          filters,
+          createTableIdentifier(tableName, schema)
```

Comment:
This is inconcistent with the other fragment methods; it's also unsafe.
It would perhaps be justified if you create the table identifyer once, and then reuse it throughout the `list` method, but here you create the table identifier twice!